manifest-build-tools/pushToBintray.sh

```diff
@@ -0,0 +1,213 @@
+#!/bin/bash
+
+############################################
+# Usage: 
+# pushToBintray.sh \
+# --user username \
+# --api_key api_key \
+# --subject subject \
+# --repo bintray_repo \
+# --package bintray_package_name \
+# --version bintray_package_version \
+# --file_path local_package_path \
+# --component "main" \
+# --distribution "trusty" \
+# --architecture "amd64,i386"
+#
+# The parameters are required:
+# user, api_key, subject, repo, package, version, file_path,
```

Comment:
How about more verbose ?

- subject:  the Bintray subject, which is either a user or an organization
- user:  bintray user name
- api_key: the Bintray API key with privilege being able to upload packages to target "subject"
-  component/distribution/architecture:  are the attributes of debian packages used to identify the apt-get source

anyway, good job and :+1: